guides/doc-Provisioning_Guide/topics/proc_installing_ipv6_environment.adoc

```diff
@@ -0,0 +1,264 @@
+
+== Requirements
+
+ifeval::["{build}" == "satellite"]
+{SmartProxy} must be installed on {RHEL} version 7.9 or higher.
++
+{Project} or {SmartProxy} can be only installed on IPv6-only systems, dualstack is not supported.
+endif::[]
+
+ifeval::["{build}" != "satellite"]
+{SmartProxy} must be installed on a system with grub2 version 2.05 or higher or system with fixes for HTTP Boot, for example CentOS version 7.9 or 8.3 or higher. For other operating systems, copy newest grub build to `/var/lib/tftpboot/grub2/grubx64.efi`.
+endif::[]
+
+The workflow requires {Project} deoloyment with IPv6 only networking, dual-stack setup has not been tested. In the examples below, IPv6 subnet `2001:db8::/64` is subject for provisioning and {ProjectServer} has IPv6 address `2001:db8::2`.
+
+[options="nowrap" subs="+quotes,attributes"]
+----
+# nmcli c add con-name eth0 type ethernet ifname eth0 \
+        ip6 2001:db8::2/64 gw6 2001:db8::1 \
+        ipv6.dns 2001:db8::1
+----
+
+Since clients which are booting from network cannot be identified by remote IPv6 address due to unmanaged DHCPv6 service, provisioning tokens must be enabled. To check that, go to Administer - Settings - Provisioning and check that Token duration is not set to zero.
+
+It is assumed, native IPv6 internet connectivity is availble. If not, it's recommended to configure a HTTP proxy in order to download RPM packages:
+
+ifeval::["{build}" == "satellite"]
+[options="nowrap" subs="+quotes,attributes"]
+----
+# cat /etc/rhsm/rhsm.conf
+proxy_hostname = myproxy.example.com
+proxy_port = 8080
+proxy_user = optional_proxy_username
+proxy_password = optional_proxy_password
+----
+endif::[]
+
+ifeval::["{build}" != "satellite"]
+----
+echo "proxy=http://[2001:db8::::1]:8888" >> /etc/yum.conf
+----
+endif::[]
+
+== Hotfix for Foreman 1.24 or Katello 3.13
+
+NOTE: This should be removed for Foreman 2.0 or Satellite 6.8.
+
+The following two installer patches are required to continue:
+
+[options="nowrap" subs="+quotes,attributes"]
+----
+pushd /usr/share/foreman-installer/modules/dns
+curl https://github.com/theforeman/puppet-dns/commit/7f5c514607f802e75081e50f1483888aca10d950.patch > dns-ip-fix.patch
+patch -p1 -f < dns-ip-fix.patch
+popd
+
+pushd /usr/share/foreman-installer/modules/foreman_proxy
+curl https://patch-diff.githubusercontent.com/raw/theforeman/puppet-foreman_proxy/pull/521.patch > dns-ip-fix.patch
+patch -p1 -f < dns-ip-fix.patch
+popd
+----
+
+Also one additional custom option must be explicitly set:
+
+[options="nowrap" subs="+quotes,attributes"]
+----
+# # https://github.com/theforeman/puppet-foreman_proxy/pull/557
+# cat >/etc/foreman-installer/custom-hiera.yaml <<EOF
+foreman_proxy::plugin::dynflow::core_listen: "'::1'"
+EOF
+----
```

Comment:
Including this one I assume @ekohl ? Dropping thanks.

Reply:
Yes, that is no longer needed.